src/_locales/sv/messages.json

```
@@ -54,5 +54,38 @@
   },
   "btnFullScreen": {
     "message": "Visa i helskärm"
+  },
+  "kbdShortcuts": {
+    "message": "Tangentbordsgenvägar"
+  },
+  "kbdMovement": {
+    "message": "Rörelsenycklar"
+  },
+  "kbdFeature": {
+    "message": "Navigationsnycklar för geografiska egenskaper"
+  },
+  "kbdPanUp": {
+    "message": "Panorera upp"
+  },
+  "kbdPanDown": {
+    "message": "Panorera ner"
+  },
+  "kbdPanLeft": {
+    "message": "Panorera till vänster"
+  },
+  "kbdPanRight": {
+    "message": "Panorera till höger"
```

```suggestion
    "message": "Panorera höger"
```

I think I prefer "Pan right", as opposed to "Pan to right".